android/app/src/main/java/app/organicmaps/car/util/Colors.java

```diff
@@ -14,6 +14,7 @@ public final class Colors
   public static final CarColor START_NAVIGATION = CarColor.GREEN;
   public static final CarColor NAVIGATION_TEMPLATE_BACKGROUND_DAY = CarColor.GREEN;
   public static final CarColor NAVIGATION_TEMPLATE_BACKGROUND_NIGHT = CarColor.DEFAULT;
+  public static final CarColor BUTTON_ACCEPT = CarColor.GREEN;
```

Comment:
Is it possible to manage colors in resources?

Reply:
No